flask_admin/model/base.py

```diff
@@ -459,6 +459,31 @@ class MyModelView(BaseModelView):
         actions endpoints are accessible.
     """
 
+    column_list_row_actions = None
+    """
+        List of row actions (instances of :class:`~flask_admin.model.template.BaseListRowAction`).
+
+        Behaviour of this field is affected by `columns_override_default_row_actions` field
+        value. By default, Flask-Admin will generate standard per-row actions (edit, delete, etc)
+        and will append custom actions from this list right after them.
+
+        For example::
+
+            from flask_admin.model.template import EndpointLinkRowAction
+
+            class MyModelView(BaseModelView):
+                column_list_row_actions = [
+                    EndpointLinkRowAction('.some_view', 'fa fa-cross'),
+                    EndpointLinkRowAction('.another_view', 'fa fa-eye-open'),
+                ]
+    """
+
+    column_override_default_row_actions = False
```

Comment:
I'm not sure this will be used much. I'd prefer to have developers do `can_view = False`, `can_edit = False`, and `can_delete = False` rather than add a new attribute for this.